util/store/store.go

```diff
@@ -0,0 +1,194 @@
+package store
+
```

Comment:
This utilities package standardizes the functions used in `x/leverage/keeper/store.go`. It's being used for incentive module right away but I plan to move leverage over to it as well, in a separate PR.